pkg/metronome/client.go

```diff
@@ -76,3 +78,26 @@ func (c *Client) Jobs(opts ...JobsOption) ([]Job, error) {
 
 	return jobs, err
 }
+
+// RunJob triggers a run of the job with a given runID right now.
+func (c *Client) RunJob(runID string) error {
+	resp, err := c.http.Post("/v1/jobs/"+runID+"/runs", "application/json", nil)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode != 201 {
```

Comment:
In general it is more readable to return early in such cases (as it avoids too many nested block levels) : 
``` go 
	if resp.StatusCode == 201 {
		return nil
	}
	if resp.StatusCode == 404 {
		return fmt.Errorf("job %s does not exist", runID)
	}
 	var apiError *dcos.Error
	if err := json.NewDecoder(resp.Body).Decode(&apiError); err != nil {
		return err
	}
	return apiError
```

A switch statement might even be more suited than the above here.